cmd/otel-allocator/config/config.go

```diff
@@ -38,12 +39,14 @@ import (
 
 const DefaultResyncTime = 5 * time.Minute
 const DefaultConfigFilePath string = "/conf/targetallocator.yaml"
+const DefaultCRScrapeInterval model.Duration = model.Duration(time.Second * 30)
 
 type Config struct {
 	LabelSelector          map[string]string  `yaml:"label_selector,omitempty"`
 	Config                 *promconfig.Config `yaml:"config"`
 	AllocationStrategy     *string            `yaml:"allocation_strategy,omitempty"`
 	FilterStrategy         *string            `yaml:"filter_strategy,omitempty"`
+	CRScrapeInterval       model.Duration     `yaml:"cr_scrape_interval,omitempty"`
```

Comment:
I thought some more about this over the weekend, and I think this may be better if we make an embedded CR config resource and that's where we'll be able to then move the incorrect Pod and Service monitor selectors to. Does that seem reasonable to you? ([this issue](https://github.com/open-telemetry/opentelemetry-operator/issues/1907))

Reply:
Sure, what should we name it? I'm ok with `PrometheusCR` for consistency, but no strong opinions. Also, should we allow enabling the functionality via the config file (currently it can only be done via a CLI flag)?

Also, do you want to move the current selector attributes, as opposed to creating new ones with the right semantics?

Reply:
PrometheusCR makes sense. And I'm not sure I have an opinion on where the enabling should happen... i trust your judgement. And let's keep the current selector attributes where they are now, and then in a follow up make the new ones in the `PrometheusCR`, deprecating the current ones.

Reply:
Done. I left the CLIConf alone for now, it's not entirely clear what the logic should be there, and I'd rather do it in a separate PR without these other changes in the way.